evol/population.py

```diff
@@ -217,6 +222,18 @@ def mutate(self, func, probability=1.0, **kwargs) -> 'Population':
             individual.mutate(func, probability=probability, **kwargs)
         return self
 
+    def log(self, **kwargs) -> 'Population':
```

Comment:
Should we allow the user to _not_ evaluate before logging? We could have a flag that allows that

Reply:
This may add confusion. What would be the base behaviour? If an individual is not evaluated we do not log it?